http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -213,6 +215,18 @@ END
       gsutil cp ${CUSTOM_ENV_VARS_CONFIG_URI} /etc
     fi
 
+    # Install RStudio license file, if specified
+    if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
+      STAT_EXIT_CODE=0
+      gsutil -q stat ${RSTUDIO_LICENSE_FILE} || STAT_EXIT_CODE=$?
```

Comment:
Tolerate the license not being there, so this PR can be merged ahead of fc-develop

Reply:
can we add a TODO for removing this once fc-develop PR is merged?

Reply:
Will do.. was thinking if there's ever a case where we still want to support RStudio _without_ a license (in which case we'd want to keep this). But I don't think so.